### PR TITLE
.NET 8

### PR DESCRIPTION
**Pull request by KazWolfe**

### CODE REVIEW

File: Dalamud/Plugin/Internal/PluginManager.cs
Comment (marzent) [inline Dalamud/Plugin/Internal/PluginManager.cs:847]: Do we want to remove these `// NET8 CHORE` comments, before merging?

Comment (KazWolfe) [reply Dalamud/Plugin/Internal/PluginManager.cs:847]: These look like they're killing off MonoMod, so it's not a blocker to release. We definitely will need to do a cleanup phase at some point though - will defer to Goat if he wants to do that before merging or after.

Comment (goaaats) [reply Dalamud/Plugin/Internal/PluginManager.cs:847]: We should do a pass for API 10.

**Review by goaaats — APPROVED** [review]

**Pull request merged**
